sklearn/manifold/tests/test_locally_linear.py

```diff
@@ -26,7 +26,8 @@ def test_barycenter_kneighbors_graph():
     # check that columns sum to one
     assert_array_almost_equal(np.sum(A.toarray(), 1), np.ones(3))
     pred = np.dot(A.toarray(), X)
-    assert linalg.norm(pred - X) / X.shape[0] < 1
+    assert linalg.norm((pred - X),
+                       check_finite=False) / X.shape[0] < 1
```

Comment:
For tests, I think we should keep `check_finite=True` to make sure our tests do not generate nonfinite values.